### PR TITLE
feat(graph): add schema for app iap subscription

### PR DESCRIPTION
**Pull request by bcgodev**
@HiIamAlanOu This PR adds the schema for apple iap subscription. Please kindly review it and let me know your thought.

### CODE REVIEW

**Review by HiIamAlanOu — APPROVED** [review]

**Comment by bcgodev** [comment]
@HiIamAlanOu I update the schema as we discussed earlier.
@nickhsine Please take a look as it's meat to used by both apple and android devices.

File: graph/member/mutation.graphql [hunk 1]
```diff
@@ -13,6 +13,13 @@ type Mutation {
   Nested query is not allowed in the mutation.
   """
   updatemember(id: ID!, data: memberUpdateInput!): memberInfo
+
+  """
+  It creates a subscription with subscriptionAppCreateInfo, set a new order number, validate the info, and set the amount/currency coresponding to the **info**.
+  It will also creates a paypemt record binding to the subscription.
```
Comment (nickhsine) [inline graph/member/mutation.graphql:19]: `paypemt` -> `payment`

File: graph/member/type.graphql [hunk 2]
```diff
@@ -3127,6 +3132,13 @@ input subscriptionOneTimeCreateInfo {
   returnToPath: String!
 }
 
+input subscriptionAppCreateInfo {
+  firebaseId: String!
+  productId: String!
+  source: createSubscriptionAppSourceType!
+  verificationData: String!
```
Comment (nickhsine) [inline graph/member/type.graphql:3139]: What will the `verificationData` look like ?

Comment (HiIamAlanOu) [reply graph/member/type.graphql:3139]: `verificationData` is `purchaseToken`

File: graph/member/mutation.graphql (hunk 1 above)
Comment (bcgodev) [reply graph/member/mutation.graphql:19]: Thanks! Great eye.

**Pull request merged**